lib/Service/ApplicationService.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @author Baptiste Fotia <baptiste.fotia@arawa.fr> for Arawa (https://arawa.fr)
+ *
+ * GroupFolders
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\GroupFolders\Service;
+
+use OCA\GroupFolders\AppInfo\Application;
+use OCP\App\IAppManager;
+
+class ApplicationService {
+
+    /** @var IAppManager */
+    private $appManager;
+
+    public function __construct(IAppManager $appManager)
+    {
```

Comment:
```suggestion
    private IAppManager $appManager;

    public function __construct(IAppManager $appManager) {
```

Reply:
I fixed it !

https://github.com/nextcloud/groupfolders/pull/2072/commits/3a2c342c4666031220df7256f9224f488e91042d